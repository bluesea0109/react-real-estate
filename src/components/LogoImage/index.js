import React from 'react';
import { Image } from 'semantic-ui-react';

export default () => <Image size="small" src={require('./logo_bm.svg')} alt="Brivity Marketer Logo" />;
