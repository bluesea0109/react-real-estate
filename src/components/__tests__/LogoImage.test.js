import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import LogoImage from '../LogoImage';

test('renders correctly', () => {
  const tree = renderer.create(<LogoImage />);
  expect(tree).toMatchSnapshot();
});
