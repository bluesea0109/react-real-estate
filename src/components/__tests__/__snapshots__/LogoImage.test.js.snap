// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<img
  alt="Brivity Marketer Logo"
  className="ui small image"
  src={
    Object {
      "ReactComponent": Object {
        "$$typeof": Symbol(react.forward_ref),
        "render": [Function],
      },
      "__esModule": true,
      "default": "logo_bm.svg",
    }
  }
/>
`;
